ros_controllers/src/joint_trajectory_controller.cpp

```diff
@@ -22,133 +22,126 @@
 #include <vector>
 
 #include "builtin_interfaces/msg/time.hpp"
+
 #include "lifecycle_msgs/msg/transition.hpp"
 #include "lifecycle_msgs/msg/state.hpp"
+
 #include "rclcpp/time.hpp"
+
 #include "rclcpp_lifecycle/state.hpp"
+
 #include "rcutils/logging_macros.h"
+
 #include "trajectory_msgs/msg/joint_trajectory.hpp"
 #include "trajectory_msgs/msg/joint_trajectory_point.hpp"
 
 namespace ros_controllers
 {
 
 using namespace std::chrono_literals;
+using controller_interface::CONTROLLER_INTERFACE_RET_SUCCESS;
+using lifecycle_msgs::msg::State;
 
-namespace
-{
+JointTrajectoryController::JointTrajectoryController()
+: controller_interface::ControllerInterface(),
+  joint_names_({}),
```

Comment:
This could just be `joint_names_{}` in `joint_trajectory_controller.hpp` (along with `write_op_names_{}`).

Reply:
we don't have any default initializers within this code base, so I would place these style changes for another set of PRs.